torch/onnx/_internal/exporter.py

```diff
@@ -78,7 +79,25 @@ def __init__(
         self.logger = logger
 
 
-class ResolvedExportOptions:
+class ResolvedExportOptions(ExportOptions):
+    """Consolidates `ExportOptions` with default values.
+    All unspecified options from `ExportOptions` are assigned a default value.
+    This is an internal class and its API may be changed at any time without notice.
+    """
+
+    # Public attributes MUST be redefined below without ``Optional[]`` from ``ExportOptions``
+    opset_version: int
+    dynamic_shapes: bool
+    op_level_debug: bool
+    logger: logging.Logger
+
+    # Private only attributes
+    decomposition_table: Dict[torch._ops.OpOverload, Callable]
+    """Dictionary with all ATen operators supported by the exporter."""
+
+    fx_tracer: FXGraphExtractor
```

Comment:
nit: prepend `_` if private?

Reply:
Keptit without `_` because it is used outside `exporter.py` scope, although inside `_internal` folder. This is the same criteria we used to change `_ResolvedExportOptions` to `ResolvedExportOptions`. Both are defined at `exporter.py` and used in different files.

Reply:
SG! In that case I would just refine the comment to clarify it is not a private field to the class.

Reply:
The comment above states "# Private only attributes" but the fields do not have private names. This creates confusion for ourselves.

Reply:
Perhaps just change that to `# Internal attributes` or `# Exporter internal attributes` etc.? Using `internal` removes that ambiguity because "private" tends to mean a very specify scope

Reply:
Thank you for your suggestion, but this is a won't fix. Too minor